astropy/coordinates/builtin_frames/skyoffset.py

```diff
@@ -177,3 +183,20 @@ def represent_as(self, base, s='base', in_frame_units=False):
         data = super().represent_as(base, s, in_frame_units=in_frame_units)
         self._set_skyoffset_data_lon_wrap_angle(data)
         return data
+
+
+def __getattr__(name):
+    if name in __all__ and name in globals():
+        return globals()[name]
+
+    if not name.startswith('SkyOffset'):
+        raise AttributeError(f"Module {__name__!r} has no attribute {name!r}.")
```

Comment:
lower-case "module" to match the standard error message

Reply:
I agree that https://github.com/astropy/astropy/pull/13305 is a cleaner alternative. Just re-opening this because it's not actually resolved. Presumably a moot point if this PR is closed in favor of #13305